lib/db/reasoning.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * @module stardogjs.db.reasoning
+ */
+import { getFetchDispatcher } from '../request-utils';
+import { RequestHeader, ContentType, RequestMethod } from '../constants';
+import { BaseDatabaseOptions, BaseDatabaseOptionsWithGraphUri } from '../types';
+
+// TODO: Confirm whether this is really necessary.
```

Comment:
Looks like maybe not, since you're able to set it in dispatchFetch. Based on the name and my memory I'm pretty sure I wrote it, but can't remember why any more.